buildSrc/src/main/groovy/org.apache.fineract.dependencies.gradle

```diff
@@ -235,6 +236,8 @@ dependencyManagement {
         //v42.7.5: performance issue: https://github.com/pgjdbc/pgjdbc/issues/3511#issuecomment-2637277977
         dependency 'org.postgresql:postgresql:42.7.4'
 
+        dependency 'com.mysql:mysql-connector-j:9.2.0'
```

Comment:
What it the difference between this and `mysql:mysql-connector-java:8.0.33`?
However i am still unsure whether we can use mysql-connector as dependency... i acknowledge seems we already doing so having it in `fineract-provider/build.gradle` as driver...